openfl/callbacks/callback_list.py

```diff
@@ -52,6 +52,7 @@ def _add_default_callbacks(self, add_memory_profiler, add_metric_writer):
         """Add default callbacks to callbacks list if not already present."""
         self._memory_profiler = None
         self._metric_writer = None
+        self._model_saver = None
```

Comment:
Since we are now appending callbacks during init, this may no longer be required